collector/lib/SysdigService.h

```diff
@@ -89,6 +93,7 @@ class SysdigService : public Sysdig {
 
   void AddSignalHandler(std::unique_ptr<SignalHandler> signal_handler);
 
+  mutable std::mutex sysdig_mutex_;
```

Comment:
The Falco project has endeavored to remove `sysdig` from the codebase after donation to the CNCF, please rename this to remove `sysdig`, something like `libsinsp_mutex` maybe?

Reply:
Renamed as suggested.

Do you think that we should rename the whole SysdigService as well ?